dashboard/src/app/ide/ide.controller.js

```diff
@@ -81,6 +81,14 @@ class IdeCtrl {
 
       let promise = cheWorkspace.fetchWorkspaces();
 
+      if ($routeParams.showLogs) {
+        // remove action from path
+        $location.url('/ide/' + this.selectedWorkspaceName, false);
```

Comment:
I don't think it could be hardcoded. (the value /ide) 
it should be a property like https://github.com/eclipse/che/blob/master/dashboard/src/assets/branding/product.json#L6